zubhub_frontend/zubhub/package.json

```diff
@@ -32,7 +32,7 @@
     "react-quill": "^2.0.0",
     "react-redux": "^7.2.2",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^5.0.1",
```

Comment:
Is this really needed? We don't want to make major changes in versions for now

Reply:
Okay, I completely understand

Reply:
Okay, I completely understand

Reply:
Okay, I completely understand